af/oometa/006-test-lists.install.sql

```diff
@@ -0,0 +1,42 @@
+BEGIN;
+
+select _v.register_patch( '006-test-lists', ARRAY[ '005-repeated-report' ], NULL );
+
+CREATE SEQUENCE IF NOT EXISTS cat_no_seq;
+
+CREATE TABLE IF NOT EXISTS url_categories
+(
+    cat_no INT NOT NULL default nextval('cat_no_seq') PRIMARY KEY,
```

Comment:
FYI: the reason for me to create SEQUENCEs as separate DB objects was to reuse same SEQUENCE as a default value for different tables (e.g. temporary table and data table).